src/lang/modifyAst/tagManagement.ts

```diff
@@ -194,12 +193,15 @@ export function createTagExpressions(
 
         // Modify the tag based on selectionType
         if (artifact.type === 'sweepEdge' && artifact.subType === 'opposite') {
-          tagCall = createCallExpressionStdLib('getOppositeEdge', [tagCall])
```

Comment:
No kw equivalent here?

Reply:
Oops, that's an oversight, I don't know how I let that through.